slave-base-ubuntu/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:12.04.5
+FROM ubuntu:16.04
```

Comment:
Ran into problems running Jenkins Plugins Installer on the new slave. Issues could be fixed by using a newer version of Python. The easiest way I could think of to do this was to change the base image to Ubuntu 16.04. Slave now has Python 2.7.12 instead of 2.7.3 and JPI runs as epxected.

Are there any objections with changing the base image to use Ubuntu 16.04 as other slave image of based of this image too? May this cause other issues?